README.md

```diff
@@ -67,53 +67,74 @@ make DEBUG=1 trusted_os && make qemu
 00:00:02 TA starting ssh server (SHA256:eeMIwwN/zw1ov1BvO6sW3wtYi463sq+oLgKhmAew1WE) at 10.0.0.1:22
 ```
 
-Trusted Applet authentication
-=============================
-
-To maintain the chain of trust the OS performs trusted applet authentication
-before loading it, to this end the `APPLET_PUBLIC_KEY` and `APPLET_PRIVATE_KEY`
-environment variables must be set to the path of either
-[signify](https://man.openbsd.org/signify) or
-[minisign](https://jedisct1.github.io/minisign/) keys, while compiling.
+## Trusted Applet authentication
 
-Example key generation (signify):
+To maintain the chain of trust the Trusted Applet must be signed and logged.
+To this end, two [note](https://pkg.go.dev/golang.org/x/mod/sumdb/note) signing keys
+must be generated.
 
-```
-signify -G -n -p armored-witness-applet.pub -s armored-witness-applet.sec
+```bash
+$ go run github.com/transparency-dev/serverless-log/cmd/generate_keys@HEAD \
+  --key_name="DEV-TrustedApplet" \
+  --out_priv=armored-witness-applet.sec \
+  --out_pub=armored-witness-applet.pub
 ```
 
-Example key generation (minisign):
+The corresponding public key files will be built into the Trusted OS to verify the Applet.
 
-```
-minisign -G -p armored-witness-applet.pub -s armored-witness-applet.sec
+## Firmware transparency
+
+All ArmoredWitness firmware artefacts need to be added to a firmware transparency log.
+
+The provided `Makefile` has support for maintaining a local firmware transparency
+log on disk. This is intended to be used for development only.
+
+In order to use this functionality, a log key pair can be generated with the
+following command:
+
+```bash
+$ go run github.com/transparency-dev/serverless-log/cmd/generate_keys@HEAD \
+  --key_name="DEV-Log" \
+  --out_priv=armored-witness-log.sec \
+  --out_pub=armored-witness-log.pub
 ```
 
-Building the compiler
-=====================
+## Building and executing on ARM targets
+
+### Building the compiler
```

Comment:
Consider steering people down the default path of just downloading the compiler rather than compiling it?